EarlGrey/Action/GREYActions.m

```diff
@@ -389,18 +389,24 @@ + (void)grey_setText:(NSString *)text onWebElement:(id)element {
       }
       BOOL elementIsUIControl = [element isKindOfClass:[UIControl class]];
       BOOL elementIsUITextField = [element isKindOfClass:[UITextField class]];
+      BOOL elementIsUITextView = [element isKindOfClass:[UITextView class]];
 
       // Did begin editing notifications.
       if (elementIsUIControl) {
         [element sendActionsForControlEvents:UIControlEventEditingDidBegin];
       }
-
       if (elementIsUITextField) {
         NSNotification *notification =
             [NSNotification notificationWithName:UITextFieldTextDidBeginEditingNotification
                                           object:element];
         [NSNotificationCenter.defaultCenter postNotification:notification];
       }
+      if (elementIsUITextView) {
+        NSNotification *notification =
+            [NSNotification notificationWithName:UITextViewTextDidBeginEditingNotification
+                                      object:element];
```

Comment:
please also align color (:) with the one above.

Reply:
@khandpur  done